cdc/capture/capture.go

```diff
@@ -454,6 +454,16 @@ func (c *Capture) runEtcdWorker(
 	timerInterval time.Duration,
 	role string,
 ) error {
+	if role == "owner" {
+		pc, err := pdutil.NewPlacementClient(ctx, c.PDClient)
+		if err != nil {
+			log.Error("Fail to create placement client in owner", zap.Error(err))
+			// return a nil error to trigger another retry of the upper-level function
+			return nil
+		}
+		go pc.Run(pdutil.DefaultCheckInterval)
+		defer pc.Close()
+	}
```

Comment:
`runEtcdWorker` has a clear meaning, I prefer to put the placement setter logic in `campaignOwner` as before. such as

```go
pc, err := pdutil.NewPlacementClient(ctx, c.PDClient)
if err != nil {
    return err
}
go pc.Run(pdutil.DefaultCheckInterval)
err = c.runEtcdWorker(ownerCtx, owner, orchestrator.NewGlobalState(), ownerFlushInterval, "owner")
c.setOwner(nil)
pc.Close()           
```

Reply:
Done, PTAL!